features/step_definitions/sign_up_steps.rb

```diff
@@ -0,0 +1,44 @@
+Given("the following user exists") do |table|
+  table.hashes.each do |user|
+    User.create(user)
+  end
+end
+
+When("I fill in {string} with {string}") do |field, value|
+  fill_in field, with: value
+end
+
+When("I click button {string}") do |element|
+  click_button element
+end
+
+When("I press the {string} button") do |sign_up|
+  click_link sign_up
+end
+
+Then("I should be redirected to the {string} page") do |page_name|
+  expect(page.current_path).to eq page_path_from(page_name)
+end
+
+Then("I should be on the {string} page") do |page_name|
+  visit page_path_from(page_name)
```

Comment:
This should be an assertion instead of an action...
```ruby
expect(page.current_path).to eq page_path_from(page_name)
```

This means you can then merge this step with the one at [Line 19](https://github.com/CraftAcademy/rails_messaging_october_17_sterik/pull/5/files#diff-85646afdd6ded23c3dd7b9a369b3c333R19) instead

to become

```ruby
Then("I should be (redirected to)/on the {string} page") do |page_name|
  expect(page.current_path).to eq page_path_from(page_name)
end
```